airflow/executors/celery_executor.py

```diff
@@ -63,13 +62,9 @@ def execute_command(command_to_exec: CommandType) -> None:
     """Executes command."""
     log = LoggingMixin().log
     log.info("Executing command in Celery: %s", command_to_exec)
-    env = os.environ.copy()
     try:
-        subprocess.check_call(command_to_exec, stderr=subprocess.STDOUT,
```

Comment:
This will mean that Environment Variables won't be preserved.

Reply:
That would also mean all the Connections created via Environment variables won't work unless LocalTaskJob copies the current vars

Reply:
When `os.fork` is used in `StandardTaskRunner` it is fine. But for cases where `CAN_FORK=False`, this would fail

Reply:
Environment variables will be preserved - since it's the same process :)